string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,29 @@
+/* exported isVowel */
+
+/**
+ * PSEUDOCODE:
+ * Define a function called 'isVowel' that takes in one parameter 'char'.
+ *  Declare variable 'vowels' and set value to array of vowels.
+ *  Declare variable 'vowelComparison' and set value to false.
+ *  Begin a loop that iterates through all characters in 'vowels' where
+ *    For each index in the string:
+ *      If the character matches the lowercased char,
+ *        reassign vowelComparison to true and break.
+ *  Return vowelComparison.
+ **/
+
+/**
+ * CODE IMPLEMENTATION:
+ */
+
+function isVowel(char) {
+  var vowels = ['a', 'e', 'i', 'o', 'u'];
+  var vowelComparison = false;
+  for (var i = 0; i < vowels.length; i++) {
+    if (vowels[i] === char.toLowerCase()) {
+      vowelComparison = true;
+      break;
+    }
+  }
+  return vowelComparison;
```

Comment:
We have not covered this in class, but you can also use something called a `switch` statement here! [Here's](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) some additional reading if you're interested 
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```